mastodon/src/main/java/org/joinmastodon/android/utils/StatusFilterPredicate.java

```diff
@@ -11,43 +20,47 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-public class StatusFilterPredicate implements Predicate<Status>{
+public class StatusFilterPredicate implements Predicate<Status> {
+
+	//Hide in timelines and warn in threads
+	private final List<Filter> clientFilters = List.of(	new AltTextFilter(HIDE, HOME, PUBLIC, ACCOUNT),
+														new AltTextFilter(WARN, THREAD, NOTIFICATIONS));
```

Comment:
Please format this properly.

Reply:
I didn't find a .editorconfig and saw different formattings between different files. Is there a project wide definition?

Reply:
I didn't find a .editorconfig and saw different formattings between different files. Is there a project wide definition?

Reply:
Not really. The official app often omits spaces when assigning variables, whereas we Megalodon, and we often don't. So as long as it looks fine, it's ok. The issue here was the tab spacing and the basically empty new line.